python/GafferUI/AnimationEditor.py

```diff
@@ -443,9 +443,9 @@ def __init__( self ) :
 
 		# create editors
 		# NOTE: initial value type (e.g. int or float) determines validated value type of widget
-		self.__frameEditor = GafferUI.NumericWidget( value=int(0) )
-		self.__valueEditor = GafferUI.NumericWidget( value=float(0) )
-		self.__interpolationEditor = GafferUI.MenuButton()
+		self.__frameEditor = GafferUI.NumericWidget( value=int(0), toolTip="# Frame\n\nEdit frame of currently selected keys." )
+		self.__valueEditor = GafferUI.NumericWidget( value=float(0), toolTip="# Value\n\nEdit value of currently selected keys." )
```

Comment:
We lost our resident UI/docs expert who would have been able to weight in more authoritatively, but we tend not to use the imperative ("Edit") in tooltips. The typical form is something that could be prefixed with "This" or "This is" and still be a valid sentence that describes the thing being represented in the UI. So in this case "The frame of the currently selected keys" (the editability of the field already implies that it can be edited). In the NodeEditor the tooltip also goes on the label, so it might be good to do that here too.